Setup/UpgradeData.php

```diff
@@ -629,21 +628,93 @@ public function upgrade(ModuleDataSetupInterface $setup, ModuleContextInterface
             $connection->insert($table, ['path' => $insuranceBelgium, 'value' => 1]);
         }
 
-        if (version_compare($context->getVersion(), '4.3.0', '<=')) {
+        if (version_compare($context->getVersion(), '4.4.0', '<=')) {
             $setup->startSetup();
+            $eavSetup
+                ->removeAttribute(Product::ENTITY, 'myparcel_digital_stamp')
+                ->removeAttribute(Product::ENTITY, 'myparcel_classification')
+                ->removeAttribute(Product::ENTITY, 'myparcel_disable_checkout');
 
-            $ids = ['myparcel_disable_checkout', 'myparcel_classification', 'myparcel_digital_stamp'];
-
-            foreach ($ids as $id) {
-                foreach (['searchable', 'filterable'] as $field) {
-                    $eavSetup->updateAttribute(
-                        Product::ENTITY,
-                        $id,
-                        $field,
-                        false
-                    );
-                }
-            }
+            $eavSetup->addAttribute(
+                Product::ENTITY,
+                'myparcel_digital_stamp',
+                [
```

Comment:
kunnen er niet wat dingen uit deze gigantische arrays in een `$defaults` oid?

Reply:
waarom is deze resolved? ik zie nog steeds die grote arrays die nogal op elkaar lijken 👀